ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,65 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function EinExprs.einexpr(tn::ITensorNetwork{T}; optimizer::EinExprs.Optimizer) where {T}
```

Comment:
```suggestion
function EinExprs.einexpr(tn::ITensorNetwork; optimizer::EinExprs.Optimizer)
```
and then use `vertextype(tn)` instead of `T` in the body of the function. Generally I try to avoid using `where` when possible since it can usually be replaced by function calls which make code more generic, though we haven't moved all of the code in `ITensorNetworks` over to that style yet.